components/StandardListItem/StandardListItem.jsx

```diff
@@ -0,0 +1,48 @@
+import {PropTypes, Component } from 'react'
+import React from 'react'
+
+require('./StandardListItemStyles.scss')
+ // showIcon: true -> render the icon
+ // showLabel: true -> render the label
+ // imgSrc: source for the icon
+ // labelText: The text for the label
+ // placeIcon: defines the position of the icon. Either: top | left | right.  Default to top
+
+class StandardListItem extends Component {
+  constructor(props) {
+    super(props)
+    this.placeIcon = (this.props.placeIcon && this.props.placeIcon !== '') ? this.props.placeIcon : 'top'
```

Comment:
@k9101 can we handle this with default properties,  just like you are doing for showIcon and showLabel?